layers/meta-balena-5x-owa/recipes-core/pmsrv/pmsrv_1.00.bb

```diff
@@ -0,0 +1,23 @@
+DESCRIPTION = "Binary for pmsrv.service"
+SECTION = "Binaries"
+DEPENDS = ""
+LICENSE = "CLOSED"
+FILESEXTRAPATHS:prepend := "${THISDIR}/src:"
+
+SRC_URI = "file://pmsrv \
+           file://pmsrv.json \ 
+"
+
+S = "${WORKDIR}"
+
+INSANE_SKIP:${PN} += "already-stripped"
+
+do_install() {
+    # create the /usr/bin folder in the rootfs with default permissions
+    install -d ${D}${bindir}
+    install -m 0755 -d ${D}${sysconfdir}
+
+    # install the application into the /usr/bin folder with default permissions
+    install ${WORKDIR}/pmsrv ${D}${bindir}
+    install ${WORKDIR}/pmsrv.json ${D}${sysconfdir}
+}
```

Comment:
Let's add this `pmsrv` as it needs to refresh the uC watchdog - balenaOS probably should be offering a hook to run this type of services but there is nothing at the moment.